notebooks/lensed_quasar_population_with_variability.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>I don't follow what these lines of code are doing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I was hust looking at properties of the lens. These lines are not needed. So, I removed them.</p>